src/Microsoft.AspNetCore.Mvc.Core/DefaultApiConventions.cs

```diff
@@ -2,30 +2,48 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
```

Comment:
This isn't a namespace users should need. What's here?

Reply:
All of the attributes are defined in it. It could presumably go in the `ApiExplorer` namespace.